app/javascript/react/components/CoffeeShopShowContainer.js

```diff
@@ -67,7 +73,7 @@ const CoffeeShopShowContainer = (props) => {
       <p>{coffeeShop.description}</p>
       <div>
         <h2>Reviews</h2>
-        <ReviewFormContainer addNewReview={addNewReview} />
+        <ReviewFormContainer addNewReview={addNewReview} setErrors={setErrors} errors={errors} />
```

Comment:
up to you, but you can potentially conditionally render the form whether of not the current user is signed in. In that case, you would need to track the current user in state which could be serialized in your SHOW fetch request